docs/docsite/rst/community/communication.rst

```diff
@@ -16,57 +17,79 @@ Mailing list information
-- ``#ansible-container`` - For discussions on Ansible Container.
-- ``#ansible-jboss`` - Channel for discussing JBoss and Ansible related things.
-- ``#ansible-network`` - Channel for discussing Network and Ansible related things.
-- ``#ansible-news`` - Channel for discussing Ansible Communication & News related things.
-- ``#ansible-vmware`` - For discussions on Ansible & VMware.
-- ``#ansible-windows`` - For discussions on Ansible & Windows.
+We have various Working Groups in the Community which you can get involved with.
+
+
+- ``#ansible-aws`` `AWS Working Group <https://github.com/ansible/community/tree/master/group-aws>`_
+- ``#ansible-awx`` `AWX Working Group <https://github.com/ansible/community/tree/master/group-awx>`_
+- ``#ansible-azure`` `Azure Working Group <https://github.com/ansible/community/tree/master/group-azure>`_
+- ``#ansible-community`` `Community Working Group <https://github.com/ansible/community/tree/master/group-community>`_
+- ``#ansible-community`` `Contributor Experience Group <https://github.com/ansible/community/tree/master/group-contributor-experience>`_
+- ``#ansible-container`` `Container Working Group <https://github.com/ansible/community/tree/master/group-container>`_
+- ``#ansible-docs`` `Documentation Working Group <https://github.com/ansible/community/tree/master/group-docs>`_
+- `Docker Working Group <https://github.com/ansible/community/issues/379>`_
+- ``#ansible-galaxy`` `Galaxy Working Group <https://github.com/ansible/community/tree/master/group-galaxy>`_
+- ``#ansible-jboss`` `JBoss Working Group <https://github.com/ansible/community/tree/master/group-jboss>`_
+- ``#ansible-lightbulb`` `Lightbulb Training <https://github.com/ansible/lightbulb>`_
+- ``#ansible-linode`` `Linode Working Group <https://github.com/ansible/community/tree/master/group-linode>`_
+- ``#ansible-lockdown`` `Ansible playbook roles for security <https://ansiblelockdown.io>`_ `gh/ansible/ansible-lockdown <https://github.com/ansible/ansible-lockdown>`_
+- ``#molecule-users`` `Molecule <https://molecule.readthedocs.io>`_ designed to aid in the development and testing of Ansible roles
+- ``#ansible-network`` `Network Working Group <https://github.com/ansible/community/tree/master/group-network>`_
+- ``#ansible-devel`` `Testing Working Group <https://github.com/ansible/community/tree/master/group-testing>`_
+- ``#ansible-vmware`` `VMware Working Group <https://github.com/ansible/community/tree/master/group-vmware>`_
+- ``#ansible-windows`` `Windows Working Group <https://github.com/ansible/community/tree/master/group-windows>`_
+- ``#ansible-community`` `Ambassadors <https://github.com/ansible/ambassadors>`_
 
 
+Want to `form a new Working Group <https://github.com/ansible/community/blob/master/WORKING-GROUPS.md>`_.
+
 Language-specific channels
 --------------------------
 
 - ``#ansible-es`` - Channel for Spanish speaking Ansible community.
+- ``#ansibleu`` - Channel for the European Ansible Community.
```

Comment:
Should there be a hyphen in this name (ansible-eu)?

Reply:
nop, this is actually correct :(

Reply:
no, and single `e` as well :(